pipeline/terraform/stack/service_id_minter.tf

```diff
@@ -77,7 +77,7 @@ module "id_minter" {
 module "id_minter_topic" {
   source = "../modules/topic"
 
-  name       = "${local.namespace_hyphen}_work_id_minter"
+  name       = "${local.namespace_hyphen}_id_minter_output"
```

Comment:
Why this change?

Reply:
The old name comes from a time when we had separate ID minters for images and works; that’s no longer the case.

And although “output” is implicit from this being SNS, we do have topics that aren’t the output of any app and being explicit about that is useful.